website/deploy/docs/_print/index.html

```diff
@@ -1211,7 +1211,7 @@ <h1 id="pg-64d3292746685dfc68398bbce370e5b9">3.4 - Destroying your cluster</h1>
         
       </div>
       <div class="col-12 col-sm-4 text-center py-2 order-sm-2">
-        <small class="text-white">&copy; 2021 Cloud Sandbox All Rights Reserved</small>
+        <small class="text-white">&copy; 2021 Cloud Sandbox Authors All Rights Reserved</small>
```

Comment:
nit: Cloud **Operations** Sandbox and add period: "Cloud Operations Sandbox Authors."